### PR TITLE
Reservation checkout

### PR DESCRIPTION
**Pull request by kssampson**
-added link to valet reservation
-added link for scanner component in valet reservation
-added basic styling to valet reservation

### CODE REVIEW

File: src/App.jsx
Comment (bzw24) [inline src/App.jsx:1]: you do not need to import your components here, you've already imported them on main to link it to that endpoint (no where in the code you are using the component here)

File: src/main.jsx
Comment (bzw24) [inline src/main.jsx:1]: recommended to pull from main consistently to avoid merge conflicts.

**Review by bzw24 — CHANGES_REQUESTED** [review]
Added comments, I'm going to help resolve the conflicts before merge.

File: src/App.jsx
Comment (bzw24) [reply src/App.jsx:1]: Note: Deleted the imports to avoid conflicts

File: src/main.jsx
Comment (bzw24) [reply src/main.jsx:1]: Note: resolved the conflicts from most recent commit

**Review by bzw24 — APPROVED** [review]
Approved, but check out the comments for future reference.

**Review by stevkim — APPROVED** [review]
Note: we manually had to update the files to consolidate merge conflicts.

Reviewed as a group and approved the changes.

**Pull request merged**